ecl/eclcc/eclcc.cpp

```diff
@@ -2146,6 +2151,30 @@ void EclCompileInstance::getTargetPlatform(StringBuffer & result)
         return eclcc.getTargetPlatform(result);
 }
 
+void EclCC::appendNeverSimplifyList(const char *attribsList)
+{
+    const char * p = attribsList;
+    while (*p)
+    {
+        StringBuffer attribRegex;
+        if (*p == ',') p++;
+        for (; *p && *p != ','; p++)
+        {
+            if (*p=='/' || *p=='.' || *p=='\\')
+                attribRegex.append("\\.");
+            else
+                attribRegex.append(*p);
+        }
+        if (attribRegex.length() > 0)
+        {
+            if (neverSimplifyRegEx.length() > 0)
+                neverSimplifyRegEx.append("|");
+
+            neverSimplifyRegEx.append(attribRegex.str()).append("(\\..+)?");
```

Comment:
What is the trailing pattern for?

Reply:
It matches all sub-scopes.  For example, if the user specifies "ut", it will exclude "ut.*"

Reply:
I think automatically watching all child scopes is fine (and then no need for supporting wildcards).  The command line help should give a hint that it can also match a directory.
(Also worth commenting what the pattern does on this line.)